lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -922,8 +922,43 @@ WitnessChecker::lookupValueWitnessesViaImplementsAttr(
   }
 }
 
-SmallVector<ValueDecl *, 4> 
-WitnessChecker::lookupValueWitnesses(ValueDecl *req, bool *ignoringNames) {
+bool WitnessChecker::isMemberOperator(FuncDecl *decl, Type type) {
+  auto *DC = decl->getDeclContext();
+  auto selfNominal = DC->getSelfNominalTypeDecl();
```

Comment:
Wouldn’t it be better to use the nominal type of the conformance itself to compare against? Adoptee->getAnyNominal()